packages/block-editor/src/components/block-settings-menu/style.scss

```diff
@@ -1,59 +1,11 @@
-.block-editor-block-settings-menu__toggle .dashicon {
+.block-editor-block-settings-menu .components-dropdown-menu__toggle .dashicon {
 	transform: rotate(90deg);
 }
 
-// Popout menu
-.block-editor-block-settings-menu__popover {
-	&::before,
-	&::after {
-		margin-left: 2px;
-	}
-
-	.block-editor-block-settings-menu__content {
-		padding: ($grid-size - $border-width) 0;
-	}
-
-	.block-editor-block-settings-menu__separator {
-		margin-top: $grid-size;
-		margin-bottom: $grid-size;
-		margin-left: 0;
-		margin-right: 0;
-		border-top: $border-width solid $light-gray-500;
-
-		// Check if the separator is the last child in the node and if so, hide itself
-		&:last-child {
-			display: none;
-		}
-	}
-
-	.block-editor-block-settings-menu__title {
-		display: block;
-		padding: 6px;
-		color: $dark-gray-300;
-	}
-
-	// Menu items
-	.block-editor-block-settings-menu__control {
-		width: 100%;
-		justify-content: flex-start;
-		background: none;
-		outline: none;
-		border-radius: 0;
-		color: $dark-gray-500;
-		text-align: left;
-		cursor: pointer;
-		@include menu-style__neutral;
-
-		&:hover:not(:disabled):not([aria-disabled="true"]) {
-			@include menu-style__hover;
-		}
-
-		&:focus:not(:disabled):not([aria-disabled="true"]) {
-			@include menu-style__focus;
-		}
+.block-editor-block-settings-menu .components-icon-button.has-text svg {
+	margin-right: 0;
+}
 
-		.dashicon {
-			margin-right: 5px;
-		}
-	}
+.block-editor-block-settings-menu .components-dropdown-menu__indicator {
+	display: none;
```

Comment:
**_Needs Design Feedback_**

Note: It strikes me as worth exploring further why an indicator (downward caret) should be a default behavior if we then choose to hide it in specific circumstances. Should it be shown for block settings? Should it not be shown for any dropdown menu? Should it be a proper option of a DropdownMenu (a prop)?

What goes into the decision to say that the block settings menu shouldn't have the indicator? Is it the fact that an ellipsis is enough of an indicator in its own right to hint that there's an expanded set of actions?

Reply:
I updated the code to display this dropdown when the icon name isn't `ellipsis` which isn't great as well. This still needs to be addressed.